src/test/scripts/generate-random-data.ts

```diff
@@ -17,6 +17,7 @@ const data = new Array(DOCUMENTS_COUNT).fill(null).map((): HomeDocument => {
     const hasProperty = faker.number.int({ min: 0, max: 1 })
     const hasPropertyAreaSquared = faker.number.int({ min: 0, max: 1 })
     const areaSquared = faker.number.int({ min: 1, max: 1_000_000 })
+    const contractDate = faker.date.between({ from: '2023-01-01', to: '2023-12-31' }).toISOString()
```

Comment:
does it have to be converted to iso string? 🤔

Reply:
yes, I want it to be typed as date in elasticsearch, not as text